src/bitset.cpp

```diff
@@ -10,6 +10,8 @@
 
 #include "sugar.hpp"
 
+Bitset::Bitset() : value_(0, false) {}
```

Comment:
Would you mind telling me where it was important to have a default constructor?

Reply:
Modified functions that required default constructor.